business_requirement_deliverable/models/business.py

```diff
@@ -5,12 +5,23 @@
 from openerp.exceptions import Warning as UserError
 from openerp.exceptions import ValidationError
 from openerp.tools.translate import _
+from lxml import etree
 
 
 class BusinessRequirementResource(models.Model):
     _name = "business.requirement.resource"
     _description = "Business Requirement Resource"
 
+    @api.model
+    def default_get(self, fields):
```

Comment:
@sudhir-serpentcs Not blocking but could you explain we is need to modify `defaul_get`?